doc/SURVEYOPS/ops/index.rst

```diff
@@ -0,0 +1,12 @@
+===
+ops
+===
+
+The ops directory should be described here.
```

Comment:
Is the description of the ops directory beyond the scope of this PR?

Reply:
It is. Eddie is on the hook to describe the files in the `ops` directory. I thought it might help to include a placeholder for when Eddie is ready to take on that task.